draft-ietf-httpbis-message-signatures.md

```diff
@@ -382,7 +343,153 @@ This example shows a canonicalized value for the parameters of a given signature
   created=1618884475;expires=1618884775
 ~~~
 
-Note that an HTTP message could contain multiple signatures, but only the signature parameters used for the current signature are included in this field.
+Note that an HTTP message could contain multiple signatures, but only the signature parameters used for the current signature are included in this entry.
+
+### Method {#content-request-method}
+
+The `@method` identifier refers to the HTTP method of a request message. The value of is canonicalized by taking the value of the method as a string. Note that the method name is case-sensitive as per {{!SEMANTICS=RFC7231}}, and conventionally standardized method names are uppercase US-ASCII.
+
+For example, the following request message:
+
+~~~
+POST /path?param=value HTTP/1.1
+Host: www.example.com
+~~~
+
+Would result in the following `@method` value:
+
+~~~
+"@method": POST
+~~~
+
+### Authority {#content-request-authority}
+
+The `@authority` identifier refers to the authority component of the target of the HTTP request message. In HTTP 1.1, this is usually conveyed using the `Host` header, while in HTTP 2 and HTTP 3 it is conveyed using the `:authority` pseudo-header. The value is the fully-qualified authority component of the request, comprised of the host and, optionally, port of the request target, as a string.
```

Comment:
It'd probably be good to reference the relevant sections, to avoid ambiguity about what value to use.

Reply:
You also need to be careful about c14n here.  Is this as it appears in the protocol, verbatim, or are you expected to case-normalize and remove (or add) any explicit default port value?

The document uses structured field canonicalization, but that is incomplete when it comes to important fields like authority, which effectively has its own c14n rules.

This isn't a new issue of course, but the breakdown here makes it more obvious.